src/pages/CommitDetailPage/CommitCoverage/routes/FilesChangedTab/FilesChangedTable/FilesChangedTable.tsx

```diff
@@ -30,7 +30,7 @@ const CommitFileDiff = lazy(() => import('../shared/CommitFileDiff'))
 const columnHelper = createColumnHelper<ImpactedFileType>()
 
 const isNumericValue = (value: string) =>
-  value === 'patchPercentage' || value === 'head' || value === 'change'
+  value === 'patchPercentage' || value === 'coverage' || value === 'change'
```

Comment:
'head' was the wrong value for this - was causing weird table style for this column.